.github/workflows/coder.yaml

```diff
@@ -284,17 +284,31 @@ jobs:
             -e POSTGRES_DB=postgres \
             -e PGDATA=/tmp \
             -p 5432:5432 \
-            -d postgres:11 \
+            -d postgres:13 \
             -c shared_buffers=1GB \
-            -c max_connections=1000
+            -c max_connections=1000 \
+            -c fsync=false
           while ! pg_isready -h 127.0.0.1
           do
               echo "$(date) - waiting for database to start"
               sleep 0.5
           done
 
+      - name: Create Migration Template
+        id: migrated
+        run: |
+          DB_FROM=$(go run scripts/migrate-ci/main.go)
+          echo "::set-output name=db-from::$DB_FROM"
+          echo $DB_FROM
```

Comment:
passing this information around in environment variables is going to make it hard to reproduce this functionality locally.  Is there any way we could store it in the database itself?  Like, name the template database something deterministically generated from the migration version (e.g. `coder_template_v25`), or have a table in the main `postgres` database with mapping?

Reply:
This doesn't seem terrible to reproduce locally to me, you can just follow these CI steps. 

Seems like a deterministic way based off the migration version might be more difficult to follow or bug-prone. I could edit a migration, and it would be broken.

Reply:
well, if you can cook up a single `make` target that runs the tests like the CI, then that obviously passes the test of reproducible.  If the expectation is that people reproducing it have to open up `coder.yaml` and copy and paste stuff out of it, then that's a drag.  Maybe a `make` target that does something like

`DB_FROM=$(go run scripts/migrate-ci/main.go) && go test...`

IDK, what do you think?

Reply:
Added!